docs/Options.md

```diff
@@ -23,6 +23,7 @@ SPACESHIP_PROMPT_ORDER=(
   git           # Git section (git_branch + git_status)
   hg            # Mercurial section (hg_branch  + hg_status)
   package       # Package version
+  gradle        # Gradle section
```

Comment:
Why `maven` section is not mentioned here?

Reply:
fixed